src/components/SnazzyButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { createUseStyles } from 'react-jss';
+import { snazzyButtonStyle } from './helpers';
+
+
+interface ButtonProps {
+  children: React.ReactNode;
+}
+
+const styles = {
+  snazzyButton: {
+
+  }
+};
```

Comment:
You MUST use the imported snazzyButtonStyle here, like this: 

```suggestion
const styles = {
  snazzyButton: snazzyButtonStyle
};
```